app/src/main/java/org/kiwix/kiwixmobile/nav/destination/library/LocalLibraryFragment.kt

```diff
@@ -393,4 +413,46 @@ class LocalLibraryFragment : BaseFragment() {
   private fun navigateToLocalFileTransferFragment() {
     requireActivity().navigate(R.id.localFileTransferFragment)
   }
+
+  private fun shouldShowRationalePermission() =
```

Comment:
Why do we need permission to tell people the rationale? `shouldShowRationalePermission` -> `shouldShowPermissionRationale`